tests/dummy/app/pods/docs/quickstart/template.md

```diff
@@ -63,28 +100,30 @@ This quickstart guide will get you going with a docs site for your brand new add
     </div>
   {{/docs-snippet}}
 
-8. **Add more docs pages.** It's up to you how to structure your docs - use the Snippet, Viewer and Demo components to help you write your documentation. Let's add another page to ours: we'll add the route, link to our docs viewer, and the actual template.
+7. **Launch your docs site**. Run `ember serve` and browse to
+`localhost:4200/docs` to enjoy your brand new documentation website.
 
-  {{#docs-snippet name='quickstart-more-docs.js' title='tests/dummy/app/router.js'}}
-    this.route('docs', function() {
-      this.route('installation');
-    });
-  {{/docs-snippet}}
+8. **Create more pages.** To add more doc pages simply follow the same steps as
+used for the `Usage page in above examples`:
 
-  {{#docs-snippet name='quickstart-more-nav.hbs' title='tests/dummy/app/templates/docs.hbs'}}
-    {{#viewer.nav as |nav|}}
-      {{nav.item 'Installation' 'docs.installation'}}
-    {{/viewer.nav}}
-  {{/docs-snippet}}
+  - create a docs subroute
+  - add a corresponding navigation item to the `docs` template
+  - create a corresponding Markdown template
 
-  {{#docs-snippet name='quickstart-installation-readme.md' title='tests/dummy/app/templates/docs/installation.md'}}
-    # Installation
+## Optional
 
-    To install My Addon, run...
-  {{/docs-snippet}}
+1. **Keyboard navigation.** You may want to enable keyboard navigation for your
+docs site by adding the `DocsKeyboardShortcuts` component to your dummy
+application template. Once enabled you can use `?` to show the navigation help
+windows.
+
+  {{docs-snippet name="docs-demo-application-template.hbs" title="tests/dummy/app/templates/application.hbs"}}
 
-8. **Round out your site.**
+2. **Optional: add a favicon.** You may want to place a favicon in the
+`tests/dummy/public` folder. We recommend using using
+ [Ember CLI Favicon](https://github.com/davewasmer/ember-cli-favicon).
 
-* **Add a favicon to your dummy app.** We recommend using [Ember CLI Favicon](https://github.com/davewasmer/ember-cli-favicon).
+3. **Optional: better scrolling.** You may want to install
```

Comment:
Do these need the `Optional:` prefix since they're already under an `## Optional` heading?

Reply:
No they don't. I just forgot to remove those after I did the first one. Nice catch, just pushed an update.